### PR TITLE
Budget app

### PR DESCRIPTION
**Pull request by MICHAELKITH**
## Description

This pull request addresses the completion of the Ruby on Rails capstone project, focusing on building a mobile web application for budget management. The project follows the given requirements and guidelines, resulting in a functional and user-friendly application. The changes made in this pull request include:

- Repository setup and tools configuration.
- Implementation of models and database schema based on the provided ERD diagram.
- Integration of Devise gem for authentication and authorization.
- Design and implementation of UI pages, including sign up, log in, home page (categories page), and transactions page.
- Addition of forms and validation logic for creating categories and transactions.
- Inclusion of unit and integration tests for critical components.
- Deployment of the application and testing in a production environment.
- Creation of comprehensive documentation, including a README file and a 5-minute video demonstration.

## Changes Made

- Set up the repository and configured necessary tools.
- Created models, migrations, and associations based on the ERD diagram.
- Integrated Devise gem for user authentication and authorization.
- Developed UI pages for sign up, log in, home page, and transactions page.
- Implemented forms and validation logic for creating categories and transactions.
- Added unit and integration tests to ensure functionality and reliability.
- Deployed the application for production and performed testing.
- Created a detailed README file with instructions on setup and usage.
- Recorded a 5-minute video demonstration showcasing the project's features.

## Screenshots / Video

Please find the video demonstration of the project [here](https://www.loom.com/share/dc93ff904dfb41788dad1bf380b5b462).
## LIVE DEMO 

Please find the LIVE DEMO  of the project [here](https://mike-bpc1.onrender.com/).
## Additional Notes

- The project adheres to Ruby best practices and follows the specified Gitflow.
- The design used in the project is based on the original design idea by Gregoire Vella, with appropriate credit given in the README.

Please review and provide feedback on the code, implementation, and documentation. I believe this pull request fulfills all the project requirements and showcases my proficiency in Ruby on Rails development.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,205 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (uwadonat) [inline README.md]: - [ ] This live demo link should be pointing to the deployed application using Render, Netlify or github pages. Now it is pointing to your github page. Kindly work on it.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @MICHAELKITH,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Highlights
- No linter errors ✔️
- You are using the correct workflow ✔️
- Well-organized and clean project ✔️
- Detailed pull request message ✔️
- Good commit history ✔️
- Professional readme file  ✔️
almost


## Required Changes ♻️
_Check the comments under the review._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MICHAELKITH** [comment]
please ignore netlify linters

**Review by addod19 — APPROVED** [review]
### Hi @MICHAELKITH, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job adding the deployed live link (render) as directed by the previous code reviewer :rocket: 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**